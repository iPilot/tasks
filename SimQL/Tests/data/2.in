{
    "data": {
        "empty": {},
        "ab": 0,
        "x1": 1,
        "x2": 2,
        "y1": { "y2": { "y3": 3 } }
    },
    "queries": [ "empty", "xyz", "x1.x2", "y1.y2.z", "empty.foobar" ]
}