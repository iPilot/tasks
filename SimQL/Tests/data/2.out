empty
xyz
x1.x2
y1.y2.z
empty.foobar